resen-core/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM earthcubeingeo/resen-base:2020.1.0
+#FROM earthcubeingeo/resen-base:2020.1.0
+FROM resenbasetest:latest
```

Comment:
This needs to be changed to resen-base:2020.2.0 once we are happy with resen-base!